docs/en/stack/security/troubleshooting.asciidoc

```diff
@@ -368,6 +368,31 @@ the time on the machines within the domain is in sync.
 
 --
 
+`gss_init_sec_context() failed: An unsupported mechanism was requested`::
+
+`No credential found for: 1.2.840.113554.1.2.2 usage: Accept`::
++
+--
+
+You would usually see this error message on the client side when using `curl` to
+test {es} Kerberos setup, as the support for Kerberos Spnego is missing on the
+client due to an old version of curl. The Kerberos realm in {es} only supports
```

Comment:
Would it be correct to reword this sentence as follows?:
"...Kerberos setup. For example, these messages occur when you are using an old version of curl on the client and therefore Kerberos Spnego support is missing".